### DISCUSSION TITLE
Compatibily with surface pro x

### DISCUSSION THREAD

**Discussion by benudi** [linux-surface/linux-surface#1206] [General]
Is there any chance this will ever be possible with the pro x?

Just curious as I hate my pro X and would love to run Linux on it to give it another chance at being useful...

Thanks!

**Comment by benudi** [comment]
Nevermind, i see my question has been answered.

**Comment by qzed** [comment]
See https://github.com/linux-surface/surface-pro-x. It's still a work in
progress.

**Reply by benudi** [reply]
Yeah, just found that. Thanks!
